planning/behavior_velocity_planner/src/scene_module/stop_line/scene.cpp

```diff
@@ -64,7 +64,6 @@ bool StopLineModule::modifyPathVelocity(
 
   const auto stop_point_idx = stop_point->first;
   auto stop_pose = stop_point->second;
-  stop_pose.position.z = (stop_line_[0].z() + stop_line_[1].z()) / 2.0;
 
```

Comment:
When committing this line, I just followed the original implementation before my PR.
Currently, z position is interpolated from the path. So it makes sense to remove this line.